src/components/Table/docs/Table.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import { LiveTable } from './LiveTable';
+import { ActiveRowTable } from './ActiveRowTable';
+import { ComplexDataTable } from './ComplexDataTable';
+import { SkeletonTable } from './SkeletonTable';
+import { TablePropsTable, TableRowPropsTable, TableRowSkeletonPropsTable } from './TablePropsTable';
+import { StyledSystemLinks } from '../../../docs/StyledSystemLinks';
+
+# Table
+
+<Meta title="Components/Table" component={Table} />
+
+export const Template = args => <Table {...args} />;
+
+Tables are used to render tabular data in a basic grid, making it easier for people to scan it. Tables are comprised of cells, columns, and rows.
+
+## Properties
+
+### Table
+
+<TablePropsTable />
```

Comment:
With features that Storybook provides us with, I believe we don't need our custom props tables anymore, Storybook generates one for us with all information if we describe/comment our props interfaces correctly, define argTypes, etc.